.travis.yml

```diff
@@ -67,6 +67,11 @@ install:
   - export CGO_LDFLAGS="$CGO_LDFLAGS -L`pwd` -lrocksdb"
   - export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:`pwd`"
 
+before_script:
+  - export CHERAMI_PACKAGES=`go list ./... | grep -ve vendor | grep -ve test | sed "s#github.com/uber/cherami-server#.#"` 
```

Comment:
ALL_SRC won't work, since the tool wants either a list of packages, or a list of .go files in one package. I'll use something that gets reasonable coverage.